pkg/jobqueue/dbjobqueue/dbjobqueue.go

```diff
@@ -202,13 +219,13 @@ func (q *DBJobQueue) listen(ctx context.Context, ready chan<- struct{}) {
 		if err != nil {
 			// shutdown the listener if the context is canceled
 			if errors.Is(err, context.Canceled) {
-				logrus.Info("Shutting down the listener")
+				q.logger.Info("Shutting down the listener")
 				return
 			}
 
 			// otherwise, just log the error and continue, there might just
 			// be a temporary networking issue
-			logrus.Debugf("error waiting for notification on jobs channel: %v", err)
```

Comment:
Hmm, I wonder if we wanted to introduce a `Debug` level to `slogger`? Would it be valuable for your needs?

Reply:
I can turn this into info or introduce debug level. To my ears, temporary network issues are network issues. These should be considered as errors from the operational perspective ;-)

My concern is once we start adding levels, these can be a challenge to map onto levels from underlying implementations, therefore I was aiming for as minimum interface as possible (info/error only).

Reply:
Let's just keep them as errors right now, if this becomes too spammy, we can either modify our filters, lower the severity, or drop it. You are right, it's an error, let's treat it as such.